milli/src/update/prefix_word_pairs/snapshots/mod.rs/test_batch_bug_3043/prefix_word_pair_proximity_docids.snap

```diff
@@ -0,0 +1,4 @@
+---
+source: milli/src/update/prefix_word_pairs/mod.rs
+---
+
```

Comment:
No good reason for it indeed, it could have been an inline insta-snaphot test instead. But can we leave it for now and avoid another round of review? When/if we merge milli into meilisearch, we can build the `db_snap!` macro on top of the `meili_snap` crate. Then it would be a good time to review the format of the snapshots and the files they create :)